test/contracts/pass/cheatCodes.sol

```diff
@@ -65,8 +65,10 @@ contract CheatCodes is Test {
 
     function prove_warp_symbolic(uint128 jump) public {
         uint pre = block.timestamp;
-        hevm.warp(block.timestamp + jump);
-        assertEq(block.timestamp, pre + jump);
+        unchecked {
+          hevm.warp(block.timestamp + jump);
+          assertEq(block.timestamp, pre + jump);
+        }
```

Comment:
are these changes related to this PR?

Reply:
Nope, it's because of `--via-ir`, which is not needed.